src/main/resources/templates/token.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>token</title>
+</head>
+
+<body>
+  <h1>Refresh Token</h1>
+  <p><strong>refreshToken:</strong> <span th:text="${refreshToken}"></span></p>
+  <br />
+  <p><strong>accessToken:</strong> <span th:text="${accessToken}"></span> </p>
```

Comment:
nit: Add a `<h1 />` title for access token as well?

Reply:
Also can we add support to organization and show organization access token here?

Reply:
1. Updated in the following PR, change to `tokens`
2. Good point,  since we do not have a pre-build SDK here, it's complicated, will do it later.